form-editor-cae/src/main/java/com/tallence/formeditor/cae/validator/NumberValidator.java

```diff
@@ -32,10 +33,10 @@
 @ValidationMessage(name = "number", messageKey = MESSAGE_KEY_NUMBERFIELD_NAN)
 public class NumberValidator implements Validator<String>, SizeValidator {
 
-  private static final String MESSAGE_KEY_NUMBERFIELD_REQUIRED = "com.tallence.forms.numberfield.empty";
-  private static final String MESSAGE_KEY_NUMBERFIELD_MIN = "com.tallence.forms.numberfield.min";
-  private static final String MESSAGE_KEY_NUMBERFIELD_MAX = "com.tallence.forms.numberfield.max";
+  static final String MESSAGE_KEY_NUMBERFIELD_MIN = "com.tallence.forms.numberfield.min";
```

Comment:
Maybe the min and max values should then be BigInteger too.

Reply:
The values are configured in the studio form in struct int properties. String struct properties should be used then, but it might cause some trouble with the `form-editor-studio-plugin/src/main/joo/com/tallence/formeditor/studio/fields/NumberField.mxml` 
Would be nice, but I guess it would mean more efforts to implement

Reply:
It's probably not a real use case anyway...